### PR TITLE
submit button bug fixed

### PR DESCRIPTION
**Pull request by geekyNehal**

### CODE REVIEW

File: app/src/main/java/com/example/immadisairaj/quiz/QuizActivity.java [hunk 1]
```diff
@@ -272,6 +272,9 @@ public void onClick(DialogInterface dialog, int id) {
         });
         AlertDialog scoreDialog = builder.create();
         scoreDialog.show();
+        Button button = findViewById(R.id.submit);
+        button.setVisibility(View.INVISIBLE);
```
Comment (immadisairaj) [inline app/src/main/java/com/example/immadisairaj/quiz/QuizActivity.java]: If it becomes invisible. I press outside the dialogue box and there is no further way for me to go.

**Comment by immadisairaj** [comment]
There is a build error can you check that.
https://travis-ci.org/immadisairaj/Quiz/builds/467485209

**Comment by geekyNehal** [comment]
You can merge this and create one more issue for that else i could work on this.

**Comment by geekyNehal** [comment]
Will check the error for sure

File: app/src/main/java/com/example/immadisairaj/quiz/QuizActivity.java [hunk 2]
```diff
@@ -285,4 +288,8 @@ public void clickSolutions() {
         solutions.putIntegerArrayListExtra("Answers", (ArrayList<Integer>) qAndA.Answer);
         startActivity(solutions);
     }
-}
\ No newline at end of file
+    @Override
+    public void onBackPressed() {
+        Toast.makeText(this,"Back Press is not allowed",Toast.LENGTH_LONG).show();
```
Comment (immadisairaj) [inline app/src/main/java/com/example/immadisairaj/quiz/QuizActivity.java]: The toast is not imported.

**Comment by geekyNehal** [comment]
I have resolved the error please accept this pr.

**Pull request merged**